pkgs/top-level/all-packages.nix

```diff
@@ -1953,7 +1953,9 @@ with pkgs;
 
   node-glob = callPackage ../tools/misc/node-glob { };
 
-  nominatim = callPackage ../servers/nominatim { };
+  nominatim = callPackage ../servers/nominatim {
+    postgresql = postgresql_12;
```

Comment:
but the default postgres version is 15 already. Why do we need to overwrite it?

Reply:
That will probably be a breaking change.

Reply:
I agree that if we change the postgresql version we should do it in another PR. There has been upstream updates since, and I think we should do that change in a version update instead